composer.json

```diff
@@ -59,6 +59,8 @@
 		"saschaegerer/phpstan-typo3": "^1.9.1",
 		"seld/jsonlint": "^1.10.0",
 		"squizlabs/php_codesniffer": "^3.8.0",
+		"symfony/console": "^5 || ^6.4",
+		"symfony/translation": "^5 || ^6.4",
```

Comment:
```suggestion
		"symfony/translation": "^5.4 || ^6.4",
```

Reply:
We also should add these two packages to the Dependabot configuration so Dependabot won't updates these dependencies and hence drop the multi-version dependency.